Source/WebCore/dom/Position.cpp

```diff
@@ -337,12 +337,12 @@ Position::AnchorType Position::anchorTypeForLegacyEditingPosition(Node* anchorNo
 }
 
 // FIXME: This method is confusing (does it return anchorNode() or containerNode()?) and should be renamed or removed
-Element* Position::element() const
+RefPtr<Element> Position::anchorElementAncestor() const
 {
-    Node* node = anchorNode();
+    RefPtr node = anchorNode();
     while (node && !is<Element>(*node))
         node = node->parentNode();
-    return downcast<Element>(node);
+    return downcast<Element>(node.get());
```

Comment:
`downcast<Element>(WTFMove(node))` to avoid ref counting churn.

Reply:
Fixed.